src/components/FormAlertWithSubmitButton.js

```diff
@@ -35,6 +35,11 @@ const propTypes = {
 
     /** Should the button be enabled when offline */
     enabledWhenOffline: PropTypes.bool,
+
+    /** Temporarily disable press on enter, this will be removed when all instances of
+     * submit event handler is replaced with FormSubmit component
```

Comment:
Please explain the prop's work here. This comment is not suited here. You can move this to Form line 236.